File: src/main/java/ar/edu/um/tesoreria/rest/model/SpoterData.java
/**
 * 
 */
package ar.edu.um.tesoreria.rest.model;

import java.io.Serializable;
import java.math.BigDecimal;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@Entity
@Table
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor
@AllArgsConstructor
public class SpoterData extends Auditable implements Serializable {
	/**
	* 
	*/
	private static final long serialVersionUID = 211490393735165122L;

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long spoterDataId;

	private BigDecimal personaId;
	private Integer documentoId;
	private String apellido = "";
	private String nombre = "";
	private Integer facultadId;
	private Integer geograficaId;
	private Integer lectivoId;
	private Integer planId;
	private Integer carreraId;
	private String emailPersonal = "";
	private String celular = "";
	private Byte status = 0;
	private String message = "";
	private Integer tipoChequeraId;
	private Long chequeraSerieId;
	private Integer alternativaId;

}

File: src/main/java/um/tesoreria/rest/kotlin/model/SpoterData.kt
package um.tesoreria.rest.kotlin.model

import um.tesoreria.rest.model.Auditable
import jakarta.persistence.Entity
import jakarta.persistence.Table
import jakarta.persistence.Id
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import java.math.BigDecimal

@Entity
@Table
data class SpoterData(

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	var spoterDataId: Long? = null,

	var personaId: BigDecimal? = null,
	var documentoId: Int? = null,
	var apellido: String = "",
	var nombre: String = "",
	var facultadId: Int? = null,
	var geograficaId: Int? = null,
	var lectivoId: Int? = null,
	var planId: Int? = null,
	var carreraId: Int? = null,
	var emailPersonal: String = "",
	var celular: String = "",
	var status: Byte = 0,
	var message: String = "",
	var tipoChequeraId: Int? = null,
	var chequeraSerieId: Long? = null,
	var alternativaId: Int? = null


) : Auditable()